app/src/components/ArtifactTimeline.tsx

```diff
@@ -0,0 +1,112 @@
+import { Center, ScrollArea, Space, Text, Timeline } from '@mantine/core';
+import { Message } from 'tabler-icons-react';
+import { useEffect, useState } from 'react';
+import ArtifactComments from './ArtifactComments';
+
+interface ArtifactCommentProps {
+  date: string | undefined;
+  body: string | undefined;
+  author: string | undefined;
+}
+
+interface ExtensionArray {
+  extensions: undefined | fhir4.Extension[];
+}
+
+interface commentProps {
+  author: string | undefined;
+  date: string | undefined;
+  body: string | undefined;
+  type: string | undefined;
+}
```

Comment:
Looks like this is being used as a type, so maybe it would be better to define it as such?